Resources/views/backend/config/view/form/document_paypal_unified.js

```diff
@@ -17,9 +17,21 @@ Ext.define('Shopware.apps.Config.view.form.DocumentPaypalUnified', {
      */
     getFormItems: function() {
         var me = this,
-            data = me.callParent(arguments);
+            formItems = me.callParent(arguments);
 
-        data[1].items.push({
+        var elementFieldSetIndex = -1;
+        Ext.Array.each(formItems, function (item, index) {
+            if (item.name && item.name === 'elementFieldSet') {
```

Comment:
If `item.name` is `undefined`, it will never be equal to `elementFieldSet`. Hence the first condition is redundant.

Reply:
I think it should have been just `item` instead.

Reply:
I've seen cases in the wild, where items were null